### PR TITLE
Change Dashboard

### LINKED ISSUE #123: Change Main Dashboard Page
## Description
Change it so we no longer have two duplicate pages.

## Required Files
- dashboard.py

## User Story
As a user I need to dashboard layout to be clear so I can understand and navigate it easily.

### PR DESCRIPTION
**Pull request by annalisev**
Make pages for each of the games and one for the overall data, this should make navigation easier between pages.

Closes #123

### CODE REVIEW

**Review by shindym — DISMISSED** [review]
Good idea to have separate summary pages!

File: dashboard/Home.py
Comment (hhm970) [inline dashboard/Home.py:1]: Nice use of dividing dashboard  pages into individual scripts

**Review by hhm970 — DISMISSED** [review]
Looks great to me!

**Review by shindym — APPROVED** [review]
Conflicts resolved 🥇

**Pull request merged**